src/main/java/com/couchbase/lite/android/AndroidSQLiteStorageEngine.java

```diff
@@ -174,4 +184,55 @@ public String toString() {
                 "database=" + Integer.toHexString(System.identityHashCode(database)) +
                 "}";
     }
+
+    private static synchronized void loadICU4C(android.content.Context context, File workingDir) {
+        boolean systemICUFileExists = new File("/system/usr/icu/icudt53l.dat").exists();
+        String icuRootPath = systemICUFileExists ? "/system/usr" : workingDir.getAbsolutePath();
+        SQLiteJsonCollator.setICURoot(icuRootPath);
+        if(!systemICUFileExists){
+            loadICUData(context, workingDir);
+        }
+        SQLiteJsonCollator.setLocale(Locale.getDefault().toString());
```

Comment:
I have seen that in unit test before calling setLocale(), releaseICU() is called. Should it be the same thing here given that the database can be closed and reopened?

I'm curious that on an android device, if a user change the locale while he is using the app, how can the locale in the collator be updated?